datetime.fifthtry.site/index.ftd

```diff
@@ -41,28 +43,38 @@ margin: $ds.spaces.vertical-gap.medium
 -- ds.heading-small: datetime.now(diff) -> datetime.datetime:
 -- ds.column:
 align-content: left
-	-- ds.copy-regular: Returns the time as an i64 unix epoch string with nanoseconds precision in the UTC timezone. Optional parameter of `diff` which returns the time from now `diff` minutes later.
+	-- ds.copy-regular: Returns a datetime record with the current date and time in UTC. The date is stored as YYYYMMDD and the time as HHMMSSmmmnnnnnnnnn (to nanosecond precision, nanoseconds padded with zeros). Optional parameter `diff` returns the time from now `diff` minutes later.
 	-- ds.code:
 	lang: ftd
 	
 	\-- datetime.datetime current: $datetime.now()
 	-- end: ds.code
 -- end: ds.column
 
+-- ds.heading-small: datetime.combine(dt: datetime) -> integer:
+-- ds.column:
+align-content: left
+	-- ds.copy-regular: Combines a datetime record's date and time fields into a single integer representing the Unix epoch in nanoseconds (UTC). This is useful for precise time calculations or storage.
+	-- ds.code:
+	lang: ftd
+	
+	\-- integer combined: $datetime.combine(dt = $current)
+	-- end: ds.code
+-- end: ds.column
 
 -- ds.heading-small: datetime.fmt(dt: datetime, ft: string) -> string:
```

Comment:
When should such technical stuff, you should wrap them using `backtick`, e.g. `date time.fmt()`, it uses fixed width font, and stands out a little bit.